### ISSUE TITLE
Conversion rate in the top left

### ISSUE THREAD

**Issue by HarveyK1** [Kvan7/Exiled-Exchange-2#142]
## Parse Error
***Put name of item here, and in the title***

### Item Text
***Please provide the item's description***
- Can be copied with `ctrl`+`shift`+`c`
- Can also be copied from below the error message

right now the main currency is div to exalts/ exalts to div but it says div to chaos orbs in the top left
![image](https://github.com/user-attachments/assets/b6916773-3841-4535-b47a-31641cb75ea4)

**Comment by Kvan7** [comment]
Yeah, this is related to #28, I just haven't gotten around to removing all the calls yet.

This is calling poe ninja and is just showing the poe1 conversion rates still. Since there isn't an api that has conversion rates for poe2 this will just be disabled at some point

**Comment by HarveyK1** [comment]
Ah okay so there cant be a conversion rates tab for poe2 since there isnt a api for it?

**Comment by Kvan7** [comment]
Yeah, right now that is the case. There will be ones available eventually most likely. But also maybe not since the currency exchange is in the game now? I would end up guessing that there will probably still be one but it may be inaccurate due to the different exchange rates between using the bulk and using currency exchange in game. This also brings up if this should be included in the overlay if most people will just use currency exchange in game anyway.

Back to the apis I do know of a few that are getting started but I will not add them in without explicit permission, and not until their apis are fully developed. I don't think that they need their servers getting slammed while they are just trying to get it up and running. :)
